src/tequila/quantumchemistry/orbital_optimizer.py

```diff
@@ -217,8 +262,17 @@ def kernel(self, h1, h2, *args, **kwargs):
                 optimizer_arguments = self.vqe_solver_arguments["optimizer_arguments"]
             if self.silent is not None and "silent" not in optimizer_arguments:
                 optimizer_arguments["silent"] = True
+            
+            if not self.history:
+                if self.vqe_solver_arguments["initial_values"] is None:
```

Comment:
tests fail here. The reason is the following: 
self.vqe_solver_arguments can itself be None, or the "initial_values" entry might miss.

Would work like this:
```python
if (self.vqe_solver_arguments is not None) and ("initial_values" not in self.vqe_solver_arguments):
...
```